deploy/roles/gocd/meta/main.yaml

```diff
@@ -1,8 +1,7 @@
 ---
 dependencies:
   - {role: python, when: "install_python and custom_repo"}
-  #- {role: python-build, when: "not custom_repo"}
-  - {role: python27-scl, when: "not custom_repo"}
+  - {role: python-build, when: "not custom_repo"}
```

Comment:
@stirlingv We don't want to undo this change, can you update your code to keep python27-scl, and remove python-build?

Reply:
Fixed! henceforth indicated by "👍"